docs/section5/Rule5-41.md

```diff
@@ -0,0 +1,46 @@
+
+# Envelope - Rule 5-41
+
+**Rule ID:** 5-41
+**Rule Description:** Opaque roof surfaces that are not regulated (not part of opaque building envelope) must be modeled with the same thermal emittance and solar reflectance in the baseline as in the proposed design. 
+**Appendix G Section:** Section G3.1-5 Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
```

Comment:
I think the terminology is B_RMD now. @weilixu can you please confirm.

Reply:
yes, that is correct,